mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -74,14 +76,61 @@
 )
 
 
+class UploadArtifactProgressBar:
+    def __init__(self, iterable, file) -> None:
+        self.iterable = iterable
+        self.file_size = os.path.getsize(file)
+        self.pbar = None
+
+        if MLFLOW_ARTIFACTS_PROGRESS_BAR_ENABLED:
+            try:
+                from tqdm.notebook import tqdm
+
+                self.pbar = tqdm(
+                    total=self.file_size,
+                    unit="iB",
+                    unit_scale=True,
+                    unit_divisor=1024,
+                    desc=f"Uploading file {file}",
+                    miniters=1,
+                )
+            except ImportError:
+                _logger.warning(
+                    "module not found: tqdm. To enable progress bar for artifacts, "
+                    "install with `pip install tqdm`"
+                )
```

Comment:
I think this message is displayed every time this class is instantiated.

Reply:
Yes 🤔

Reply:
Can we remove the warning?